system_tasks.yml

```diff
@@ -44,7 +44,7 @@
 - name: Fetch piecewise
   git: repo=https://github.com/opentechinstitute/piecewise.git 
        dest=/opt/piecewise.git/
-       version=import-angular-app
+       version=update-ndt-test
```

Comment:
This should be master when merged right? the previous PR breaks the task because of the version issue.